cloudify_agent/installer/__init__.py

```diff
@@ -329,6 +332,20 @@ def create_custom_env_file_on_target(self, environment):
     def move(self, source, target):
         shutil.move(source, target)
 
+    def _get_certificate_file(self):
+        cert_file = self.cloudify_agent['local_rest_cert_file']
+
+        if not os.path.exists(cert_file):
+            cert_content = self.cloudify_agent.get('rest_cert_content')
+
+            if not cert_content:
+                return None
+
+            with open(cert_file) as f:
+                f.write(cert_content.decode('string-escape'))
```

Comment:
this has something to do with the way we propagate the certificate content from the manager blueprint. specifically, we pass this as an environment variable to the management worker systemd environment configuration file and it is escaped there. @noakup can probably elaborate.